### PR TITLE
Portfolio-Mobile menu

### PR DESCRIPTION
**Pull request by Kayonga99**
## In this milestone, the following were implemented:
- No linter errors.
- Use correct git-flow.
- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// portfolio-mobile menu
```
Comment (Mre55) [inline main.js:1]: As the project requires the mobile menu should appear and disappear when the user clicks the hamburger button or when chooses an option from the menus. Kindly, fix the scrollable behavior of the mobile menu it should be fixed. You may use position `fixed` only for the `pop-up` class.

## Refer to the video below 👇 

https://user-images.githubusercontent.com/34578631/162697222-f0177c45-3d72-4783-a815-a70477575e0d.mov

To stick to the design you can use background color something like this `background: #1127e5f2;` or you can use background `filter: blur(...)`.

Comment (Mre55) [inline main.js:1]: Also, kindly fix the typo here:

image

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @Kayonga99,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 
_Check the comments under the review._




## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# INVALID Code Review Request :stop_sign: 

Hi @Kayonga99, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

While you have done well in this project, you missed a crucial aspect: **collaborating!**
Very well that you did it by yourself, but collaborating brings a lot of advantages. 
We learn from each other a lot, and while you knew how to handle this milestone, your partners could learn from you. As well, it gives you and your partners the fabulous opportunity to develop your soft skills!

- [ ] As it is a project requirement, kindly, bring your partners together and explain to them how you have done your project and ask them to bring improvements and commits to this milestone as well!
**Until then, your project won't be reviewed any more**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy **COLLABORATING!** 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
```

File: main.js (hunk 1 above)
Comment (MeqdamAlqudah) [inline main.js:1]: - [ ] As the last reviewer requested Please fix the popup menu weird behavior when the user scrolls down 

image

one of the ways to solve it is to add overflow: none; to the body when the user opens the menu and  reverse it when the user closes the menu 

image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @Kayonga99  

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by aimemalaika — APPROVED** [review]
Hi @Kayonga99, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**